app/static/js/drawing.js

```diff
@@ -1,3 +1,36 @@
+// Display modal when window loads
+window.addEventListener('load', () => {
+    let wordCatModal = new bootstrap.Modal('#word-category-modal');
+    wordCatModal.show();
+    document.querySelectorAll('.category-btn').forEach(button => {
+        button.addEventListener("click", () => {
+            const category = button.getAttribute('data-category');
+            fetchWord(category, wordCatModal);
+        });
+    });
+});
+
+// Fetch a random word from the chosen category
+function fetchWord(category, modal) {
+    const url = '/get-random-word?' + new URLSearchParams({ category: category }).toString()
+    fetch(url)
+        .then(response => response.json())
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

you can use a template string  as well (just for knowledge sharing). Not mandatory.